### DISCUSSION TITLE
Рисование полигонов (drawing + editor)

### DISCUSSION THREAD

**Discussion by d3mchenko** [yandex-maps-unofficial/vue-yandex-maps#337] [Q&A (2.x)]
Добрый день. Реализации рисования полигонов нет во второй версии? Придётся юзать первую?

**Comment by daniluk4000** [comment]
Добрый день. Нативных методов нет, но ничего не мешает вам сделать это самостоятельно, по аналогии с [deflate](https://yandex-maps-unofficial.github.io/vue-yandex-maps/examples/objects/line.html)

Но нативных нет( И это не сделали Яндекс на своей стороне
